ui/src/components/App.js

```diff
@@ -10,7 +9,7 @@ class App extends React.Component {
       <Router>
         <div className='App'>
           <Switch>
-            <Route exact path="/" component={LandingPage} />
+            <Route exact path="/" component={AppDetailedView} />
```

Comment:
This is just temporary so the AppDetailedView can be easily seen & tested. Will be reverted before PR is merged